Missing env.GitVersionBranch on master

Error on `./dobi.sh build`:

```
[job: cplusplus-version] gen/gitversion/ Start
Generation complete.
[job: cplusplus-version] gen/gitversion/ Done
[alias:run bb-cplusplus-version] Done
[alias:run version] Done
[ERROR] a value is required for variable "env.GitVersionBranch"
```

--- a/builder/dobi.yaml
+++ b/builder/dobi.yaml
@@ -4,7 +4,7 @@
 image=image-cplusplus-builder:
   image: elbb/bb-cplusplus-builder
   context: builder
-  tags: ["{env.GitVersionBranch}"]
+  tags: ["{env.GitVersion_BranchVersion}"]
 
 # ===================================================
 # alias

--- a/runtime/dobi.yaml
+++ b/runtime/dobi.yaml
@@ -4,7 +4,7 @@
 image=image-cplusplus-runtime:
   image: elbb/bb-cplusplus-runtime
   context: runtime
-  tags: ["{env.GitVersionBranch}"]
+  tags: ["{env.GitVersion_BranchVersion}"]
 
 # ===================================================
 # alias

